pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -7023,3 +7023,91 @@ func TestImportJobEventLogging(t *testing.T) {
 		string(jobs.StatusRunning)}
 	backupccl.CheckEmittedEvents(t, expectedStatus, beforeSecondImport.UnixNano(), jobID, "import", "IMPORT")
 }
+
+func TestImportDefautIntSizeSetting(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	ctx := context.Background()
+	baseDir, cleanup := testutils.TempDir(t)
+	defer cleanup()
+	tc := testcluster.StartTestCluster(
+		t, 1, base.TestClusterArgs{ServerArgs: base.TestServerArgs{ExternalIODir: baseDir}})
+	defer tc.Stopper().Stop(ctx)
+	conn := tc.Conns[0]
+	sqlDB := sqlutils.MakeSQLRunner(conn)
+
+	intSizes := []int32{4, 8}
+
+	data := `
+CREATE TABLE default_int (
+    a INTEGER PRIMARY KEY,
+    b INT
+);
+`
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == "GET" {
+			_, _ = w.Write([]byte(data))
+		}
+	}))
+	defer srv.Close()
+
+	tests := []struct {
+		name             string
+		query            string
+		nonBundledFormat bool
+	}{
+		{
+			name:  "import pgdump",
+			query: `IMPORT PGDUMP ($1)`,
+		},
+		{
+			name:  "import table from pgdump",
+			query: `IMPORT TABLE default_int FROM PGDUMP ($1)`,
+		},
+		{
+			name:  "import mysqldump",
+			query: `IMPORT MYSQLDUMP ($1)`,
+		},
+		{
+			name:  "import table from mysqldump",
+			query: `IMPORT TABLE default_int FROM MYSQLDUMP ($1)`,
+		},
+		{
+			name: "non-bundled csv format",
+			query: `
+IMPORT TABLE default_int (
+    a INTEGER PRIMARY KEY,
+    b INT
+)
+CSV DATA ($1)
+`,
+			nonBundledFormat: true,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			for _, defaultIntSize := range intSizes {
```

Comment:
if you move this loop header outside the t.Run closure then we can have more descriptive test names incase one of them fails.

The name would be `fmt.Sprintf("%s_intsize=%d", test.name, defaultIntSize)`

Reply:
Done.